tests/unistd/unlink.c

```diff
@@ -92,10 +98,11 @@ void test() {
   err = unlink("dir-readonly");
   assert(err == -1);
 
-  // emscripten uses 'musl' what is an implementation of the standard library for Linux-based systems
+// emscripten uses 'musl' what is an implementation of the standard library
+// for Linux-based systems
```

Comment:
It would be nice if we could avoid making changes to unrelated comments to avoid polluting `git blame` results.